cdk/handlers/raster_handler.py

```diff
@@ -0,0 +1,42 @@
+"""AWS Lambda handler."""
+
+import asyncio
+import logging
+import os
+
+from eoapi.raster.utils import get_secret_dict
+
+# Update postgres env variables before importing titiler.pgstac.main
+pgstac_secret_arn = os.environ["PGSTAC_SECRET_ARN"]
+
+secret = get_secret_dict(pgstac_secret_arn)
+os.environ.update(
+    {
+        "postgres_host": secret["host"],
+        "postgres_dbname": secret["dbname"],
+        "postgres_user": secret["username"],
+        "postgres_pass": secret["password"],
+        "postgres_port": str(secret["port"]),
+    }
+)
```

Comment:
We could improve performance of the whole application if we set these environment variables on the Lambda itself. I don't know how slow it is to read the secret values, but it is something we have to do on every application startup. That's what I have seen in [`eoapi-devseed`](https://github.com/developmentseed/eoapi-devseed/blob/main/infrastructure/app.py#L135-L149), not sure if there would be a security concern with putting the db credentials as environment variables in the Lambda.

Reply:
Makes sense, security best practices prefers secrets manager or something equivalent but we're not granting external accounts access to view our lambda configurations, so this seems fine

Reply:
I will file an issue and we can decide if it's worth changing later.